images/s6_assets/init/nginx

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-if [ "${PULP_HTTPS}" = "true" ]; then
+if [[ "${PULP_HTTPS}" = "true" || "${PULP_HTTPS}" = "True" ]]; then
```

Comment:
```suggestion
if [ "${PULP_HTTPS,,}" = "true" ]; then
```
I am +1 for making the matching less strict.
But you should probably not define your environment variables with bool in the first place.
Yaml does not like Norway...